### PR TITLE
meaning of true rating wasn't obvious

### PR DESCRIPTION
**Pull request by raam86**
looks like it's an accepted term in the source code

### CODE REVIEW

**Comment by NicolasHug** [comment]
I fail to see what isn't clear here?

The true rating is opposed to the estimated rating. It's pretty standard. See e.g. in [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score) where the convention is to use `y_true` and `y_pred`.

**Comment by raam86** [comment]
well google searches won’t come up with a meaningful explanation and I don’t know enough to map your example from scikit learn to your single parameters. 
is true rating the rating the user has already rated this item for?
why do you pass 4 in the demo?

**Comment by NicolasHug** [comment]
> is true rating the rating the user has already rated this item for?

yes

>  why do you pass 4 in the demo?

Because you need to pass the true ratings if you eventually want to compute accuracy metrics like RMSE. But you don't have to, it's an optional parameter.

**Comment by NicolasHug** [comment]
I'll close the issue if you don't mind @raam86
